lib/WWW/WTF/Testcase/Config.pm

```diff
@@ -0,0 +1,71 @@
+package WWW::WTF::Testcase::Config;
+use Moose;
+use File::Util;
+use Config::Merged;
+use Getopt::Long;
+use common::sense;
+
+has 'config_directory' => (
+    is          => 'ro',
+    isa         => 'Str',
+    required    => 1,
+);
+
+has 'config' => (
+    is      => 'ro',
+    isa     => 'HashRef',
+    lazy    => 1,
+    default => sub { {}; },
+);
+
+sub _extend_config_from_directory {
+    my ($self, $path) = @_;
+
+    my @config_files = ();
+
+    File::Util->new->list_dir(
```

Comment:
maybe we can use a unified way for searching files (I use File::Find::Rule in lib/WWW/WTF/HTTPResource/Types/PDF.pm)